docs/downloads/automation-library/integrations/terraform/review_new_module.cm

```diff
@@ -0,0 +1,40 @@
+# -*- mode: yaml -*-
+
+manifest:
+  version: 1.0
+
+{% set misslist = [] %}
+{% for pattern in terraform %}
+{% if (newfilesinpr | match(term=pattern) | nope) %}
+{% set misslist = misslist + [pattern+' '] %}
+{% endif %}
+{% endfor %} 
+
+automations:
+  review_new_terraform_module:
+    if: 
+      - {{misslist | match(regex=r/.*/) | some}}
```

Comment:
This should also check if the PR includes a main.tf file that is new because we only want this to trigger for the creation of new modules. [This automation](https://docs.gitstream.cm/integrations/jsdoc/#enforce-jsdoc-for-new-javascript-classes) has an `is.new` custom expression that you can adapt for this. You'll need a single condition that checks if any of the files are a main.tf that is also new.

Reply:
Got it. That actually makes sense. I will have a look on this